ChangeLog.d/windows_pragma_lib_bcrypt

```diff
@@ -0,0 +1,2 @@
+Changes
+   * Add pragma bcrypt.lib to entropy_poll.c for msvc builds
```

Comment:
This describes the change from a developer's perspective, but changelog are from users. If I understand correctly, the goal is to resolve a build failure? So something like

> When building with MSVC, it is no longer necessary to link with bcrypt.lib explicitly.

Please correct if this isn't what the change does. I'm not familiar with Windows.

Reply:
Yes, this is a good description of the change. The pragma directive tells MSVC to link with `bcrypt.lib` so it doesn't have to be done explicitly.